src/pages/index.js

```diff
@@ -1,35 +1,95 @@
 import "./index.css";
 import Card from "../components/Card.js";
 import FormValidator from "../components/FormValidator.js";
-import {
-  initialCards,
-  selectors,
-  formValidationOptions,
-} from "../utils/constants.js";
+import { selectors, formValidationOptions } from "../utils/constants.js";
 import Section from "../components/Section.js";
 import PopupWithForm from "../components/PopupWithForm.js";
 import PopupWithImage from "../components/PopupWithImage.js";
 import UserInfo from "../components/UserInfo.js";
+import PopupConfirm from "../components/PopupWithConfirm.js";
+import Api from "../components/Api.js";
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  authToken: "5782927d-9e6f-49bd-bfea-d56552d67f06",
+});
+
+function handleDeleteModal(card) {
+  deletePopup.open();
+  deletePopup.setSubmitFunction(() => {
+    api.toDeleteCard(card._id).then(() => {
+      deletePopup.close();
+      card.removeCard();
+    });
```

Comment:
You need to `catch` possible errors at the very **end** of any server request.

**Please, correct it everywhere in the code**